### ISSUE TITLE
[autocrud] Create semantic association between grid and form

### ISSUE THREAD

**Issue by sissbruecker** [vaadin/hilla#1666]
For accessibility it should be clear that selecting something something in the grid edits that item in the form. Technically the grid "controls" the form, and there should be an aria attribute to reflect that association.
